draft-ietf-httpapi-deprecation-header.md

```diff
@@ -118,7 +125,7 @@ In this example the linked content provides additional information about depreca
 
 The following example uses the same link header field, but also announces a deprecation date using a Deprecation header field:
```

Comment:
since this says "announces", maybe use a date in the future, at least relative to the date when the spec may get published?

Reply:
on the other hand, further down it does say "has been deprecated". maybe tweak the "announces" wording here?

Reply:
ok now?